CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 
 <!-- next version -->
 
+## v1.0.0-rcv0011/v0.77.0
```

Comment:
This should probably only show v0.77.0

Reply:
Why do you think so? This is what we will show in the release. Maybe repeating the stable version is good to highlight the latest version?

Reply:
I would find it confusing as an end user if `v1.0.0-rcv0011` is called out on a release that doesn't include any changes for that version. Then I would have to go back until the earliest mention of that version to find what i'm looking for

Reply:
This is hard.  I agree that listing the same version multiple places make it harder to find the actual changes associated with it, but also having the changelog synced with the Release also makes sense.  

I trend towards dropping the unchanged RC candidate from the changelog so that the changelog stays exactly accurate with what changed in that release.

Reply:
Ok, sounds good. Let's drop the stable version here. I'll do it manually for now, and will update the automation after

Reply:
Ok, I'll update the automation rather than dealing with the updating this branch https://github.com/open-telemetry/opentelemetry-collector/pull/7640